doc/source/doc-style/docstrings.rst

```diff
@@ -143,17 +143,17 @@ Here is an example of a ``Parameters`` section for a class in PyAEDT:
     student_version : bool, optional
         Whether to enable the student version of AEDT. The default
         is ``False``.
-    
+
 The name of each parameter is followed by a space, a colon, a space, and then
 the data type. A parameter is optional if its keyword argument has a default shown
 in the function signature. For an optional parameter, the data type is followed by a
```

Comment:
```suggestion
the data type. A parameter is optional if its keyword argument displays a default
in the function signature. For an optional parameter, the data type is followed by a
```

Reply:
```suggestion
the data type. A parameter is optional if a default is defined in its signature. 
For an optional parameter, the data type is followed by a
```

In this sentence, the "keyword argument" seems misplaced to me. Not all parameters are keyword arguments, see https://docs.python.org/3/glossary.html#term-argument

Reply:
@PipKat I'm not sure if we should mix general edits to this page together with the addition of the type hint section.

Note that the changes I've made to the rest of the document are just the removal of trailing whitespace.

If the changes are small enough I guess we can do it in one go, but I wouldn't want to delay this PR with other changes.

Reply:
Feel free to go back to your original text on these lines. The number of edits are small--so I'd suggest making them.